pkgs/development/tools/tracy/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, lib, darwin, fetchFromGitHub, tbb, gtk2, glfw, pkgconfig, freetype}:
+stdenv.mkDerivation rec {
+  name    = "tracy-${version}";
+  version = "0.6.3";
+
+  buildInputs =
+    [glfw pkgconfig]
```

Comment:
pkgconfig needs to be in nativeBuildInputs.

Reply:
Why is this?

Reply:
@Mic92 See https://github.com/NixOS/nixpkgs/issues/89389